toolkit/scripts/sodiff/mariner-sodiff.sh

```diff
@@ -2,11 +2,36 @@
 
 # Required binaries:
 # rpm and dnf
```

Comment:
Does this comment still make sense after recent changes? Not sure what it's about.